templates/partials/_contributors.html

```diff
@@ -0,0 +1,68 @@
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2>Users and contributors</h2>
+    </div>
+
+    <div class="col-12">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/89680bdf-logo-ubuntu.svg" alt="Ubuntu logo">
```

Comment:
can you use the image template instead, for all these images?

Reply:
Yep - when we have the correct images and can get their heights/widths, for this I didn't want to spend time doing that for placeholder images.